experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/main.yaml

```diff
@@ -6,3 +6,8 @@
 - include: orc8r.yaml
   environment:
     AWS_DEFAULT_REGION: "{{ awsOrc8rRegion }}"
+  tags: orc8r
+- include: keys.yaml
+  environment:
+    AWS_DEFAULT_REGION: "{{ awsOrc8rRegion }}"
```

Comment:
I think keyHost and keyBoot are created in the awsAgwRegion and not in the awsOrc8rRegion. Typically, we only deploy Orc8r resources in the awsOrc8rRegion and do practically everything else in the awsAgwRegion.

Reply:
thanks for catching that !